modules/serverpod_auth/serverpod_auth_server/pubspec.yaml

```diff
@@ -7,11 +7,12 @@ version: 2.0.0-alpha.3
 repository: https://github.com/serverpod/serverpod
 
 environment:
-  sdk: '>=3.1.0 <4.0.0'
```

Comment:
Chore: Use `'` instead of `"`.

I standardized this across all projects to make these strings searchable.

Reply:
Weird are you sure this is updated everywhere? I did not change this manually.

Reply:
I see, my auto formatter in vscode is responsible for this...